test/torchaudio_unittest/functional/torchscript_consistency_impl.py

```diff
@@ -692,3 +692,21 @@ def func(tensor):
 
         tensor = torch.view_as_complex(torch.randn(2, 1025, 400, 2))
         self._assert_consistency_complex(func, tensor, test_paseudo_complex)
+
+
+class FunctionalFloat32Only(TestBaseMixin):
```

Comment:
Does the test in ``torchscript_consistency_impl`` also need ``FunctionalFloat32Only``?

Reply:
I'm not too sure what you mean here, can you elaborate?

Reply:
Yea I saw the tests in other file like auto_grad_impl use ``FunctionalFloat32Only``, so I'm wondering if we also add it here.

Reply:
Oops, It's the test itself. I'm confusing myself lol. Never mind.